catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -303,7 +306,6 @@ private Boolean userSpellcheckIsOn(QueryRequest request) {
 
   private boolean solrSpellcheckHasResults(QueryResponse solrResponse) {
     return solrResponse.getSpellCheckResponse() != null
-        && solrResponse.getResults().size() == 0
```

Comment:
❓ So this would mean that it will always run a "correctly spelled" search even if my original one produced results? If so, what would the result be if my "misspelled" word actually had more hits than the "correctly" spelled word? Perhaps the correct check here is that we only change the query if the resulting list would actually produce more results? Or, am I misreading how this would work?

Reply:
You are correct, the reason behind this change is to make the Did You Mean feature actually useful. I only discovered this when testing the feature.

So `Did You Mean` only pops up when there are 0 results in the original query. So when you click on `Did You Mean` to re-run the query, it ALWAYS returns 0 results, effectively making it useless. 

So the idea here is that for any misspelled words, always fix it and run the fixed query (unless it's so badly misspelled that it cannot be correct) and then the `Did You Mean` option will potentially be useful.

Reply:
After conversing with @jlcsmith we have decided to go forward with providing correct results only when the number of corrected results is greater than the current misspelled query results count.